main.js

```diff
@@ -173,6 +178,8 @@ var start_local_server = function() {
       serverRunning = false;
     });
     subpy.unref();
+  } else {
+    mainWindow.webContents.executeJavaScript("console.log('Unable to find OpenBazaar-Server at: '" + serverPath + "')");
```

Comment:
mainWindow isn't created until line 209, that line should be moved to the top of the file so mainWindow is available to everything that needs it.